src/hooks/use-user.tsx

```diff
@@ -0,0 +1,48 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import React, { ReactElement, useState, useEffect, useContext, createContext } from 'react';
+
+declare const fetch: any;
```

Comment:
Can you add `lib: ['dom']` to the tsconfig, then you wont need to declare fetch https://github.com/auth0/auth0-react/blob/master/tsconfig.json#L4

Reply:
Or you may need to add something to eslint https://github.com/vercel/next.js/blob/canary/examples/with-typescript-eslint-jest/.eslintrc.json#L15

Reply:
Added `"browser": true` to eslint config. BTW, the fetch polyfill comes bundled with Next.js 9.4.0+. Currently the SDK requires 9.3.6. If we're going to keep that minimum version, we need to use `node-fetch` or similar here.

Reply:
Let's go with 9.4 for min version